### PR TITLE
FORCE_COLOR & fix default environment for .env

### LINKED ISSUE #345: [FEATURE] Force colors where possible
A key motivation for #344 (PTY support) is to convince programs to print colors in logs. Full PTY support is a tall order, so until then, maybe we should do some buildpack-like magic to determine which environment variables to set to encourage programs to override their PTY detection.

For example, `FORCE_COLOR=1` for the npm chalk package used by next.js. See https://github.com/vercel/next.js/issues/11836#issuecomment-612580547

### LINKED ISSUE #347: [FEATURE] clear log scrollback
Sometimes, you want a clear/fresh slate in your log viewer so that you can clearly delineate new log messages quickly and visually. Typical terminals support clearing scroll back, often used for this purpose.

we should have a button in the GUI for this, and support typical hotkeys: cmd+k, ctrl+l

i suspect we could implement this by explicitly adding some kind of system event to the log & then handling the rendering of that specially in the GUI

reported via @finnigantime

### PR DESCRIPTION
**Pull request by brandonbloom**
Closes #345

### CODE REVIEW

File: internal/core/server/workspace.go [hunk 1]
```diff
@@ -411,29 +411,43 @@ func (ws *Workspace) DescribeEnvironment(ctx context.Context, input *api.Describ
...
-	_, err = os.Stat(envPath)
+	env := map[string]string{}
+
+	// Encourage programs to log with colors enabled.  The closest thing to a
+	// standard for this is <https://bixense.com/clicolors/#bug-reports>, but
+	// support is spotty. This may grow if there are other popular enviornment
+	// variables to include. If we grow PTY support, this may become unnecessary.
+	env["CLICOLOR"] = "1"
+	env["CLICOLOR_FORCE"] = "1"
+	env["FORCE_COLOR"] = "3" // https://github.com/chalk/chalk/tree/9d5b9a133c3f8aa9f24de283660de3f732964aaa#supportscolor
```
Comment (BenElgar) [inline internal/core/server/workspace.go:425]: This seems to force 24-bit colour support which, according to this link[0] at least, isn't supported by a number of major terminals, including the built-in MacOS terminal app. I don't know whether those terminals will gracefully downsample, just ignore the colours or violently explode.

[0] https://github.com/termstandard/colors#not-supporting-true-color

**Review by BenElgar — APPROVED** [review]
LGTM aside from for comment. I don't understand how this closes #347 though.

**Comment by brandonbloom** [comment]
Whoops! #347 was a typo. Ment to write #345. Fixed commit & description. Thanks for catching that!

File: internal/core/server/workspace.go (hunk 1 above)
Comment (brandonbloom) [reply internal/core/server/workspace.go:425]: Sorry for the delayed response here, but I just went and tested chalk directly with FORCE_COLOR set to 0, 1, 2, and 3 and printing colors in my Terminal.app locally. Seems to work? ¯\_(ツ)_/¯ 

If this is a problem on Linux or something, let me know.

**Pull request merged**